roles/infrastructure/tasks/teardown_terraform.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+# Copyright 2021 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Check if Terraform workspace directory exists
+  ansible.builtin.stat: 
+    path: "{{ infra__terraform_workspace_dir }}/infra"
+  register: workdir
```

Comment:
Where is this registered variable used?

Reply:
Good spot, this is no longer used. There are a few similar instances of this in the code that I have cleaned as part of 
287c30d.